ovos_config/models.py

```diff
@@ -64,6 +64,10 @@ def __init__(self, path):
         if path:
             self.load_local(path)
 
+    def __hash__(self):
+        LOG.debug(f"get hash for: {dict(self)}")
+        return hash(json.dumps(dict(self)))
```

Comment:
you need to sort keys in the json.dumps, otherwise the hash is not deterministic